ppr-ui/src/components/unitNotes/ExpiryDate.vue

```diff
@@ -17,7 +17,11 @@
           </label>
         </v-col>
         <v-col cols="12" sm="9" class="px-1">
-          <v-radio-group v-model="expiryDateType" column class="pt-0 mt-0" v-if="!hideContinuedExpiryDate">
+          <p v-if="hideContinuedExpiryDate" class="mb-6">
+            Date in the Future
+          </p>
+          <v-radio-group v-model="expiryDateType" column class="pt-0 mt-0" v-if="!hideContinuedExpiryDate"
+          >
```

Comment:
I think this could be a v-else at this point, and can you put the conditional statment first. Makes it a better easier to identify.

Reply:
Updated, thanks